### PR TITLE
Option to change code display format (Issue #373)

### PR DESCRIPTION
**Pull request by l-marchesi**
As discussed in Issue #373 I implemented the changes.

Got help from @LukasWillin, thanks!

### CODE REVIEW

File: app/src/main/java/com/beemdevelopment/aegis/ui/PreferencesFragment.java [hunk 1]
```diff
@@ -206,6 +206,12 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
         });
         timeoutPreference.getOnPreferenceChangeListener().onPreferenceChange(timeoutPreference, timeoutPreference.getText());*/
 
+        Preference codeDigitGroupingPreference = findPreference("pref_code_grouping");
```
Comment (alexbakker) [inline app/src/main/java/com/beemdevelopment/aegis/ui/PreferencesFragment.java:209]: The app crashes on this line for me. Shouldn't this be ``pref_code_group_size``?

File: app/src/main/java/com/beemdevelopment/aegis/ui/views/EntryHolder.java [hunk 2]
```diff
@@ -207,7 +214,8 @@ private void updateCode() {
         if (!(info instanceof SteamInfo)) {
             StringBuilder sb = new StringBuilder();
             for (int i = 0; i < otp.length(); i++) {
-                if (i != 0 && i % 3 == 0) {
+                // TODO: @Marchesi No magic!
```
[the remark this answers is not in the document]
Comment (l-marchesi) [reply app/src/main/java/com/beemdevelopment/aegis/ui/views/EntryHolder.java]: My bad, forgot to remove it - thanks for pointint it out :)

File: app/src/main/java/com/beemdevelopment/aegis/ui/PreferencesFragment.java (hunk 1 above)
Comment (l-marchesi) [reply app/src/main/java/com/beemdevelopment/aegis/ui/PreferencesFragment.java]: Yes, thank you. Decided last minute to rename it 🙈

**Comment by l-marchesi** [comment]
You're welcome Alex, sorry for some errors. I was searching in the whole repo for my comments and was sure that I've deleted them - but apparently not. 
However, now everything should be fixed, thank you for your review 😄

**Comment by alexbakker** [comment]
Thanks a lot!

**Pull request merged**